addons/l10n_ar_withholding/data/template/account.tax.group-ar_ex.csv

```diff
@@ -0,0 +1,29 @@
+id,name,country_id,l10n_ar_tribute_afip_code,name@es
+tax_group_withholding_vat,VAT Withholding,base.ar,"01",Retención IVA
```

Comment:
Why do we need tax groups per withholding?  Already, we do not use the tax groups in the code...  (as there is no tax summary on the payment)

Reply:
@maq-adhoc is this ready to be closed? As I see this is Outdated, and I think this point was actually resolved.
cc' @jco-odoo

Reply:
DONE!